src/common/Header/Header.tsx

```diff
@@ -47,7 +47,13 @@ const HeaderBtn = ({ path, alt } : {path: string, alt: string}) => {
     })),
     {
       label: (
-        <div style={{ display: "flex", justifyContent: "space-between" }}>
+        <div style={{ display: "flex", justifyContent: "space-between", marginTop:12 }}  onClick={
+          () => {
```

Comment:
@alshareef49 the same code is used by the switcher as well. please make the following changes to pass the code - review.
1. Change that div to a button since you are trying to make it clickable 
2. Since two elements use the same code, create a separate function and then call it in the reqd. places

Reply:
@Tushar-4781 thanks for your instruction,
I made some changes  to the code according to requirements.